static/app/views/projectInstall/platform.tsx

```diff
@@ -120,10 +131,81 @@ export function ProjectInstallPlatform({location, params, route, router}: Props)
     ? projects.find(proj => proj.slug === params.projectId)
     : undefined;
 
+  const {
+    data: projectAlertRules,
+    isLoading: projectAlertRulesIsLoading,
+    isError: projectAlertRulesIsError,
+  } = useApiQuery<IssueAlertRule[]>(
+    [`/projects/${organization.slug}/${project?.slug}/rules/`],
+    {
+      enabled: !!project?.slug,
+      staleTime: 0,
+    }
+  );
+
+  useEffect(() => {
+    if (!project || projectAlertRulesIsLoading || projectAlertRulesIsError) {
+      return;
+    }
+
+    if (gettingStartedWithProjectContext.project?.id === project.id) {
+      return;
+    }
+
+    const platformKey = Object.keys(platforms).find(
+      key => platforms[key].id === project.platform
+    );
+
+    if (!platformKey) {
+      return;
+    }
+
+    gettingStartedWithProjectContext.setProject({
+      id: project.id,
+      name: project.name,
+      // sometimes the team slug here can be undefined
+      teamSlug: project.team?.slug,
+      alertRules: projectAlertRules,
+      platform: {
+        ...omit(platforms[platformKey], 'id'),
+        key: platforms[platformKey].id,
+      } as OnboardingSelectedSDK,
+    });
+  }, [
+    gettingStartedWithProjectContext,
+    project,
+    projectAlertRules,
+    projectAlertRulesIsLoading,
+    projectAlertRulesIsError,
+  ]);
```

Comment:
this will make sure that always when the "getting-started" page is loaded, we populate the "gettingStartedWithProject" context